test/integration/apis/btc-core.test.ts

```diff
@@ -113,13 +92,4 @@ describe("BTCCore regtest", function () {
     afterEach(async () => {
         await api.disconnect();
     });
-
-    describe("getTxByOpcode", () => {
-        it("should return correct tx id", async () => {
-            // FIXME: generate bitcoin transaction with OP_RETURN value
```

Comment:
could you create a OP_RETURN tx here that you then check if it's included? I'd like to have that integration test because we had some major issues on the MS2 review that the transaction status caused issues.